### PR TITLE
API for Relief Camp Coordinators App

### PR DESCRIPTION
**Pull request by xaneem**
### Summarize
Provides an API for the Relief Camp Coordinators android application.

APIs for:
- Fetch list of relief camps by district
- Person registration to a camp
- Update relief camp supply requirements

Technical Changes:
- Adds django-rest-framework and jwt-auth modules
- api_urls.py and api_views.py for corresponding routes and controllers

Application:
The application is developed in a different repo; testing instructions available there
[Repo link](https://github.com/Appmaker-xyz/KeralaRescueRefugeeDB)

Testing instructions:
- Test existing features (mainly those involving login) to make sure nothing else is affected (very low chance)
- Testing the API endpoints via the application. See [app's repo](https://github.com/Appmaker-xyz/KeralaRescueRefugeeDB) for instructions
- Making sure data entered through the app appears correctly on website/DB

The app is planned to go live before noon on 19-Aug-2018.

### CODE REVIEW

**Comment by sparkymat** [comment]
Testing:

- [X] Login
- [X] Camp list
- [ ] Camps
- [ ] Persons

**Comment by sparkymat** [comment]
Once merged, let's copy the API documentation to the Wiki.

File: mainapp/api_views.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    queryset = Person.objects.filter()
+    serializer_class = PersonSerializer
+    permission_classes = (permissions.IsAuthenticated,)
+    http_method_names = ['post']
+
+    def create(self, request):
+        for data in request.data:
+            serializer = PersonSerializer(data=data)
+
+            data['age'] =  data['age'] or None
```
[the remark this answers is not in the document]
Comment (sparkymat) [reply mainapp/api_views.py]: Nevermind. Need to pass an array.

**Comment by saleehk** [comment]
@sparkymat Build add person endpoint as array for pushing multiple items at same time. Which helps offline syncs faster

**Comment by naveenpf** [comment]
@saleehk requirement are given by IT mission it is implemented.

**Pull request merged**